### PR TITLE
S2 canvas

### PR DESCRIPTION
**Pull request by DanielGardiner**
- converted wireframe into html and CSS skeleton 
- included crosshair cursor 
- added a function to generate multiple divs to form canvas 
- integrated canvas into page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <meta charset="UTF-8">
+    <script defer src="script.js"></script>
+    <title>JS-Paint</title>
+</head>
+<body>
+
+    <div class="formContainer">
+
+        <h1>Select a size</h1>
```
Comment (mporam) [inline index.html]: this form does nothing and is not part of this story. please remove it

File: canvas.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <script defer src="script.js"></script>
+    <title>JS-Paint</title>
+</head>
+<body>
+
+<div class="screen">
+
+    <div class="navbar">
+        <div class="icons">
+            <img src="https://img.icons8.com/officel/256/000000/rgb-circle-1.png">
```
Comment (mporam) [inline canvas.html]: these buttons do nothing, please remove them

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="normalize.css" type="text/css">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <meta charset="UTF-8">
+    <script defer src="script.js"></script>
```
Comment (mporam) [inline index.html]: please put your JS into a js folder

File: index.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="normalize.css" type="text/css">
+    <link rel="stylesheet" href="style.css" type="text/css">
```
Comment (mporam) [inline index.html]: please put your css into a css folder

File: style.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+html {
+    font-size: 62.5%;
+}
+
+h1 {
+    font-size: 2.5rem;
+}
+
+body {
+background-color: #bfbfbf;
```
Comment (mporam) [inline style.css]: indent

File: style.css [hunk 6]
```diff
@@ -0,0 +1,81 @@
+
```
Comment (mporam) [inline style.css]: lots of unnecessary line breaks and incorrect formatting in this document, please fix all of it

File: style.css [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    width: 80%;
+    height: auto;
+    margin: 10px 0;
+}
+
+.canvas {
+    height: 70vh;
+    width: 70vw;
+    margin: auto auto;
+    /*border: 2px solid black;*/
```
Comment (mporam) [inline style.css]: please remove unnecessary code

**Review by mporam — CHANGES_REQUESTED** [review]

**Comment by DanielGardiner** [comment]
- have made changes to syntax
- have removed unnecessary tasks from Story 2

**Review by mporam — CHANGES_REQUESTED** [review]
code changes look fine but the nav bar has disappeared and the canvas is not in the middle, its randomly floating

**Review by mporam — APPROVED** [review]

**Pull request merged**